src/osd/PGLog.cc

```diff
@@ -50,15 +50,8 @@ void PGLog::IndexedLog::trim(
   set<string>* trimmed_dups,
   eversion_t *write_from_dups)
 {
-  if (complete_to != log.end() &&
-      complete_to->version <= s) {
-    generic_derr << " bad trim to " << s << " when complete_to is "
-		 << complete_to->version
-		 << " on " << *this << dendl;
-    assert(0 == "out of order trim");
-  }
-
   assert(s <= can_rollback_to);
+  generic_dout(0) << " complete_to " << complete_to->version << dendl;
```

Comment:
can use ldout(cct) instead of generic_dout - and should be higher level

Reply:
the entire function uses generic_dout(), so left it the same for uniformity. Increased the level to 20.